src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -696,6 +696,15 @@ export default function ApplicationForm({
       }
     }
 
```

Comment:
do we need to check birthLoc.length > 100 ?

Reply:
Yes we will add that after.